test/integ/main_test.go

```diff
@@ -63,6 +64,11 @@ func TestJVS(t *testing.T) {
 	if keyRing == "" {
 		t.Fatal("Key ring must be provided using TEST_JVS_KMS_KEY_RING env variable.")
 	}
+	fireStoreProjectID := os.Getenv("TEST_JVS_FIRESTORE_PROJECT_ID")
+	if keyRing == "" {
+		t.Fatal("Firestore project id must be provided using TEST_JVS_FIRESTORE_PROJECT_ID env variable.")
+	}
+	justificationConfigFullPath := "jvs/key_config"
```

Comment:
Would this cause conflict if integ tests run in parallel?

Reply:
Each integration tests need to use their own path so that we can run them in parallel.
We have not changed the code in public api and cert rotation api, so it should be ok for now.

Reply:
Can you add a TODO?

Reply:
done